gh-actions-scripts/generate-k8s-resource-stats.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Print complete yaml of the keptn installer
+helm template ../dist/keptn-installer/keptn-*.tgz --dry-run > "$HELM_TEMPLATE"
+
+# yq: Extract container name and resource limits/requests for all resources
+# jq: Bring yq result into nice format that can be read by the tablemark-cli
+# unique_by: Remove duplicated distributor resource stats
```

Comment:
AFAIK every deployment that has a distributor will also consume memory and CPU resources - not sure why you want to remove it

Reply:
fixed